drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
```

Comment:
the `FromMap` is super opinionated about the data to be map[string]string.
That is mostly true for labels. Though this is a reasonable change for the given requirements,
we have to make sure how this change will affect the existing functionality. Especially the integration
with docker. @mrjana can you take a look @ this aspect ? Since we send the data from Docker daemon
during network create events, we just have to make sure if this change has any impact.